python/lsst/__init__.py

```diff
@@ -22,3 +22,4 @@
 
 import lsstimport, pkgutil
 __path__ = pkgutil.extend_path(__path__, __name__)
+
```

Comment:
Sorry, that's from the previous implementation and me not reviewing my own code before submission.